alembic/versions/20200809155758_add_pets.py

```diff
@@ -1,34 +1,33 @@
-"""Add Pet data
+"""add pets
 
-Revision ID: 0000
+Revision ID: 20200809155758
 Revises: 
-Create Date: 2020-06-19 10:26:19.245846
+Create Date: 2020-08-09 15:57:59.790572
 
 """
 from alembic import op
 import sqlalchemy as sa
 
 
 # revision identifiers, used by Alembic.
-revision = "0000"
+revision = '20200809155758'
 down_revision = None
 branch_labels = None
 depends_on = None
 
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.create_table(
-        "pet",
-        sa.Column("id", sa.Integer(), autoincrement=True, nullable=False),
-        sa.Column("name", sa.String(), nullable=False),
-        sa.Column("tag", sa.String(), nullable=True),
-        sa.PrimaryKeyConstraint("id"),
+    op.create_table('pet',
```

Comment:
So I feel table names should be plural. Like "pets". (This is another Rails convention that I agree with.) Does that break too many things?

Reply:
The endpoints are plural, but the tables are always singular. Same thing for the modules for instance.

Reply:
Looks like this is a standard to describe the item in a row instead of the contents of the table itself.